util/src/idf_parse.rs

```diff
@@ -0,0 +1,189 @@
+use csv::{ReaderBuilder, WriterBuilder};
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::fs::File;
+use thiserror::Error;
+
+use crate::deserialize::idf_date;
+
+pub const IDF_S3_PATH: &str = "/lard_reports/idf/";
```

Comment:
Can you also add a `const IDF_S3_LATEST: &str = "/lard_reports/idf/latest/"` or would it just be confusing? You'd need to format outside `push_to_s3` though, and I wish there was a way of concatenating consts and string slices

Reply:
I chose this one as the base, since in the parser I use it for both latest and adding a folder with the date. But then in the code you wrote for the endpoint you always end up looking at the latest folder... so maybe we do need both as consts?

Reply:
Yeah, I think it's fine to have both, maybe with a comment. Also you can call this `IDF_S3_PATH` `IDF_S3_BASEPATH`